client/my-sites/marketing/controller.js

```diff
@@ -25,6 +25,16 @@ import {
 } from 'state/sites/selectors';
 import versionCompare from 'lib/version-compare';
 
+export const redirectButtons = context => {
```

Comment:
What about renaming `redirectButtons` in to `redirectSharingButtons`? It's obviously longer but it better matches the path and name of the page.

Reply:
done